### PR TITLE
Closes #671: Charles/display version

### LINKED ISSUE #671: Display version next to pipeline

### PR DESCRIPTION
**Pull request by cdebourcy**
# Description
First pass at displaying version information

### CODE REVIEW

File: app/assets/src/components/PipelineSampleReads.jsx [hunk 1]
```diff
@@ -400,13 +400,23 @@ class PipelineSampleReads extends React.Component {
     sample_dropdown = <span>{ this.sampleInfo.name }</span>
     }
 
+    let version_display = !this.pipelineRun ? '' :
+                            !this.pipelineRun.version ? '' :
+                              !this.pipelineRun.version.pipeline ? '' :
+                                'v' + this.pipelineRun.version.pipeline
```
Comment (boris-dimitrov) [inline app/assets/src/components/PipelineSampleReads.jsx]: Pretty.

File: app/helpers/pipeline_outputs_helper.rb [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
+    pipeline_run_display["version"] = { pipeline: select_version_aspect(pipeline_run, "idseq-pipeline"),
+                                        nt: select_version_aspect(pipeline_run, "nt_k16"),
+                                        nr: select_version_aspect(pipeline_run, "nr_rapsearch") }
+    pipeline_run_display
+  end
+
+  def select_version_aspect(pipeline_run, aspect)
+    return nil unless pipeline_run.version
+    version_hash = JSON.parse(pipeline_run.version)
+    aspect_hash = version_hash.select { |item| item["name"] == aspect }[0]
```
Comment (boris-dimitrov) [inline app/helpers/pipeline_outputs_helper.rb]: Is the `[0]` guaranteed not to go out-of-bounds?

File: app/helpers/pipeline_outputs_helper.rb [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
+    return nil unless aspect_hash
+    version_key = %w[nt_k16 nr_rapsearch].include?(aspect) ? "source_version" : "version"
+    aspect_hash.key?(version_key) ? aspect_hash[version_key] : nil
```
Comment (boris-dimitrov) [inline app/helpers/pipeline_outputs_helper.rb]: To my reading of the Ruby spec, you can just write the last line simply as follows
```
aspect_hash.key(version_key)
```

File: app/helpers/pipeline_outputs_helper.rb (hunk 2 above)
Comment (cdebourcy) [reply app/helpers/pipeline_outputs_helper.rb]: `[0]` will just give nil if it is out-of-bounds and that case is then handled in the next line.
Ok, I'm adding an example of version_hash. It should actually be called version_array.

File: app/helpers/pipeline_outputs_helper.rb (hunk 3 above)
Comment (cdebourcy) [reply app/helpers/pipeline_outputs_helper.rb]: Hmm that doesn't actually work though

**Pull request merged**